index.rst

```diff
@@ -4,9 +4,326 @@ Monitor coupling in M1M3 VMS system and TMA drives together
+   :alt: VMS acceleration along y for accelerometer #2, TMA Orientation, TMA torque, TMA speed, M1M3 angular acceleration around x, y and z
+
+   VMS acceleration along y for accelerometer #2, TMA Orientation, TMA torque, TMA speed, M1M3 angular acceleration around x, y and z. The start and the end of the slew are indicated
+   by the vertical dashed lines.
+
+During the slew and between the acceleration and decelartion phase, we see a few vibration occurences detected by the VMS
+
+The following plot shows the acceleration along x, y and z axis recorded by accelerometer #2 and HP forces along the same axes.
+
+.. figure:: /_static/images/VMS-HP-2023-12-21-546.png
+   :name: VMS-HP-2023-12-21-546
+   :target: _images/VMS-HP-2023-12-21-546.png
+   :alt: VMS acceleration along x, y and z for accelerometer #2 and HP forces along x, y and z. The start and the end of the slew are indicated by the vertical dashed lines.
+
+   VMS acceleration along x, y and z for accelerometer #2 (in blue) and HP forces along x, y and z (in red). The start and the end of the slew are indicated by the vertical dashed lines.
+
+From this plot, we see that the larger HP forces are applied along the y and the z axes and we notice a correlation between the vibrations seen by the VMS along these 2 axes 
+(spikes occure at the same time).
+Given the scale of the HP forces in y and z directions (much more important than for an azimuth slew) it is impossible to detect any correlation between HP forces and VMS vibrations. 
+
+An analysis of the VMS vibration similar to the one described for the azimuth slew shows that the most prominent glitches appear when the TMA elevation is around:
+[43.4 55.7 60.0 61.3 65.7] degrees. The same analysis on another elevation-only slew (2023-12023 0- seqNum: 205) shows glitches at: [52.7 53.9 55.3 58.1 60.1] degrees. Even if from these
+2 slews, it seems that some glitches appear around the same elvation values, it is impossibe to draw any definitive conclusion with such a low statistics.
+
+
+Similarly to the azimuth analysis, the following figure shows the spectrogram of the VMS data for the 3 accelerometers and the 3 axes.
+
+.. figure:: /_static/images/spectrogram-2023-12-21-546.png
+   :name: spectrogram-2023-12-21-546
+   :target: _images/spectrogram-2023-12-21-546.png
+   :alt: Spectrograms of the signal recorded by the 3 VMS accelerometers along x, y and z
+
+   Spectrograms of the signal recorded by the 3 VMS accelerometers along x, y and z.
+
+The glitches are seen as vertical bands covering the whole frequency range and are visible in mostof the spectrograms.
+A few horizontal bands are visible, with one obvious around 40 Hz.
+
+Analysis of VMS data when the TMA is tracking
+---------------------------------------------
+
+In order to complete this study, we have selected an event where the TMA is tracking after sleing to a target position.
+
+* dayObs: 20231221
+* seqNum: 535
+* duration: 42s
```

Comment:
I always found curious that these tracking events have 42 seconds (instead of 30-ish). You can disregard this comment but just in case you had some insight.

Reply:
My understanding is that you have to track during ~2s of settle time + 2x15s exposure + 2x2s reading +2x2s for the shutter to move (not sure about this one). This is ~40s, there is still 2s missing...